src/exercises/day-3/simple-gui.rs

```diff
@@ -78,10 +85,7 @@ impl Window {
 impl Widget for Window {
     fn width(&self) -> usize {
         // ANCHOR_END: Window-width
-        std::cmp::max(
-            self.title.chars().count(),
-            self.widgets.iter().map(|w| w.width()).max().unwrap_or(0),
-        )
+        self.inner_width() + 4
```

Comment:
```suggestion
         // The `| ... |` formatting adds an additional 4 characters to the width.
        self.inner_width() + 4
```

Reply:
It took me a while to figure out which bug you're fixing... until it finally clicked :smile: